src/test/ui/structs-enums/enum-non-c-like-repr-int.rs

```diff
@@ -65,8 +65,7 @@ fn main() {
     unsafe {
         // This should be safe, because we don't match on it unless it's fully formed,
         // and it doesn't have a destructor.
-        #[allow(deprecated)]
-        let mut dest: MyEnum = mem::uninitialized();
+        let mut dest: MyEnum = mem::zeroed();
```

Comment:
Ideally there should be an argument why zero-initializing `MyEnum` is legal (also for the other similar tests).